### PR TITLE
Added docs for JsonResponse::fromJsonString

### LINKED ISSUE #7174: Add docs for "fromJsonString()"
In https://github.com/symfony/symfony/pull/19552 we added the named constructor `fromJsonString()` to create `JsonResponse` objects using already encoded JSON contents. Should we document or mention it?

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #7174.

### CODE REVIEW

**Comment by xabbuh** [comment]
👍 

Status: Reviewed

File: components/http_foundation.rst [hunk 1]
```diff
@@ -547,13 +547,22 @@ class, which can make this even easier::
...
-    $response->setData(array(
-        'data' => 123
-    ));
+    // ...
+    $response->setData(array('data' => 123));
+
+    // if the data to send is already encoded in JSON
+    $response = JsonResponse::fromJsonString('{ "data": 123 }');
+
+.. versionadded:: 3.2
```
Comment (stof) [inline components/http_foundation.rst]: wrong version

**Comment by stof** [comment]
Status: needs work

File: components/http_foundation.rst (hunk 1 above)
Comment (javiereguiluz) [inline components/http_foundation.rst]: Then this blog post is wrong too 😢  http://symfony.com/blog/new-in-symfony-3-2-dx-improvements#added-a-named-constructor-to-jsonresponse

Comment (xabbuh) [inline components/http_foundation.rst]: @javiereguiluz 3.2 is correct: https://github.com/symfony/symfony/pull/19552

**Review by HeahDude — APPROVED** [review]
👍

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request closed without merging**